src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
> But we should be consistent here with other window globals, either we define modules for all of them or none. 

Well, as we discovered above we did this for the convenience, modulization of every host objects seems strange